Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -179,6 +196,38 @@ public void Validate()
         public List<DisplayDirection> EntityBarDirections { get; set; } =
             Enumerable.Range(0, 1 + (int)Vitals.VitalCount).Select(_ => DisplayDirection.StartToEnd).ToList();
 
+        public List<string> TypewriterSounds { get; set; } = new List<string>()
+        {
+            DEFAULT_TYPEWRITER_SOUND
+        };
```

Comment:
Optional: Remove `string DEFAULT_TYPEWRITER_SOUND` and replace with

```csharp
public static List<string> DEFAULT_TYPEWRITER_SOUND => new List<char>()        {
            "octave-beep-tapped.wav"
        };
```

Reply:
done